web/src/beta/features/AccountAndWorkSpaceSetting/innerPages/AccountSetting/PasswordModal/index.tsx

```diff
@@ -0,0 +1,200 @@
+          setRegexMessage(t("Too short."));
+          break;
+        case passwordPolicy?.tooLong?.test(password ?? ""):
+          setRegexMessage(t("That is terribly long."));
+          break;
+        case passwordPolicy?.highSecurity?.test(password ?? ""):
+          setRegexMessage(t("That password is great!"));
+          break;
+        case passwordPolicy?.medSecurity?.test(password ?? ""):
+          setRegexMessage(t("That password is better."));
+          break;
+        case passwordPolicy?.lowSecurity?.test(password ?? ""):
+          setRegexMessage(t("That password is okay."));
+          break;
+        default:
+          setRegexMessage(t("That password confuses me, but might be okay."));
+          break;
+      }
+    },
+    [t, password, passwordPolicy] // eslint-disable-line react-hooks/exhaustive-deps
+  );
+
+  const handleClose = useCallback(() => {
+    setPassword("");
+    setPasswordConfirmation("");
+    onClose?.();
+  }, [onClose]);
+
+  const handleSave = useCallback(() => {
+    if (password === passwordConfirmation) {
+      updatePassword?.({ password, passwordConfirmation });
+      handleClose();
+    }
+  }, [updatePassword, handleClose, password, passwordConfirmation]);
+
+  useEffect(() => {
+    if (
+      password !== passwordConfirmation ||
+      (passwordPolicy?.highSecurity &&
+        !passwordPolicy.highSecurity.test(password)) ||
+      passwordPolicy?.tooShort?.test(password) ||
+      passwordPolicy?.tooLong?.test(password)
+    ) {
+      setDisabled(true);
+    } else {
+      setDisabled(false);
+    }
+  }, [password, passwordConfirmation, passwordPolicy]);
+
+  const isMatchPassword =
+    password !== passwordConfirmation && passwordConfirmation;
+
+  return (
+    <Modal size="small" visible={isVisible}>
+      <ModalPanel
+        title={t("Change password")}
+        onCancel={handleClose}
+        actions={[
+          <Button
+            key="Change password"
+            title={t("Change password")}
+            appearance="dangerous"
+            disabled={disabled}
+            onClick={handleSave}
+          />
+        ]}
+      >
+        <ModalContentWrapper>
+          <div>
+            <SubText>
+              <Typography size="body" weight="bold">
+                {t(
+                  `In order to protect your account, make sure your password is unique and strong.`
+                )}
+              </Typography>
+            </SubText>
+            <PasswordField direction="column">
+              <Typography size="body" weight="bold">
+                {t("New password")}
+              </Typography>
+              <TextInput
+                value={password}
+                onChange={handlePasswordChange}
+                type="password"
+              />
+              {password ? (
+                <PasswordMessage size="body" weight="bold">
+                  {regexMessage}
+                </PasswordMessage>
```

Comment:
I think it's better to have different colors for differenty type of regexMessage as well. (danger / warning / okey), an d please use the colors from theme.

Reply:
Already added color for regexMessage. Please check.
https://github.com/reearth/reearth-visualizer/pull/1155/commits/a431d2297da2511f6772fac437ec327fd69dc38a